Anlab.Mvc/Controllers/AdminController.cs

```diff
@@ -97,6 +97,8 @@ public IActionResult EditUser(string id, [Bind("FirstName,LastName,Name,Phone,Ac
         }
 
         [Authorize(Roles = RoleCodes.Admin)]
+        [HttpPost]
+        [ValidateAntiForgeryToken]
```

Comment:
@jSylvestre do you need this validate attribute since you put one on the application controller?